licensecheck.properties

```diff
@@ -0,0 +1,25 @@
+category=Languages
+name=License Check
+description=Checks Maven, Gradle and NPM project for license violations
+issueTrackerUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/issues
+devVersion=5.0.0-beta-2
+publicVersions=4.0.2
+archivedVersions=3.2.0
+
+defaults.mavenGroupId=at.porscheinformatik.sonarqube.licensecheck
+defaults.mavenArtifactId=sonarqube-licensecheck-plugin
+
+5.0.0-beta-2.description=JavaScript-only and Groovy-only projects, configuration uses default functionality
+5.0.0-beta-2.sqVersions=[8.9,LATEST]
+
+4.0.2.description=fix: create license with status disallowed
+4.0.2.sqVersions=[8.4,8.9.*]
```

Comment:
Once you drop 5.0.0, you'll mark this one as ending with `LATEST`.

Reply:
This still isn't compatible with `LATEST`. Do you really mean for this not to be marked compatible with 9.0 (announced earlier this week) and future versions?